src/panels/lovelace/hui-root.ts

```diff
@@ -572,9 +583,6 @@ class HUIRoot extends LitElement {
         }
         newSelectView = index;
       }
-
-      // Will allow to override history scroll restoration when using back button
-      setTimeout(() => scrollTo({ behavior: "auto", top: 0 }), 1);
```

Comment:
Doesn't removing this preserve the scroll? The comment implies that, I think

Reply:
It is moved [line 994 ](https://github.com/home-assistant/frontend/pull/24777/files#diff-1a6a1c83027cace21d04bf74ca202fdb6a0ac6061e759ffbbd5e0d4d13975463R994).

As we don't reload the page when switching view, the new view was keeping the scroll of the previous view. That's why we had this scroll to top function. Now, each view preserve its scroll position and reapply it when clicking back to have similar behavior than a regular page navigation.